pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -203,6 +204,7 @@ var (
 			Protocol:          istionetworking.ListenerProtocolTCP,
 		},
 		{
+			TransportProtocol: "raw_buffer",
```

Comment:
this cannot be raw buffer.. because this is not plaintext. it can be unknown TLS as well. put another way, add one more that has transport protocol tls as well.

Reply:
oh ignore. we have that already